src/Console/Command/ConfigureCommand.php

```diff
@@ -183,22 +182,9 @@ protected function writeConfiguration(array $configuration): bool
         return false;
     }
 
-    /**
-     * Make a guess to the bin dir.
-     */
     protected function guessBinDir(): string
     {
-        $defaultBinDir = $this->config->getBinDir();
-        if (!$this->composer()->hasConfiguration()) {
-            return $defaultBinDir;
-        }
-
-        $config = $this->composer()->getConfiguration();
-        if (!$config->has('bin-dir')) {
-            return $defaultBinDir;
-        }
-
-        return $config->get('bin-dir', Config::RELATIVE_PATHS);
+        return $this->composer()->getComposerFile()->getBinDir();
```

Comment:
Maybe also safer to force a relative path as in the old codebase? Not sure too what folder the old code is making the path relative